common/src/main/java/com/microsoft/identity/common/CodeMarker.java

```diff
@@ -0,0 +1,114 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.common;
+
+import android.util.Pair;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A Class containing information of a code marker which is an event in code.
+ * Marker is a string which is prefixed by scenario code and is defined in class {@link PerfConstants}
+ */
+public class CodeMarker {
+
+    private static String csvNoValue = "NA";
+    private static char csvSeparator = ',';
+
+    private String marker;
+    /* timeInMilliseconds represents time in milliseconds from the time of creation of first codemarker.
+    If timeInMilliseconds is zero(0) then it means that this is the first codemarker of the scenario.*/
+    private long timeInMilliseconds;
+    // timeStamp is the system time at the time of the capture of the codemarker.
+    private String timeStamp;
+    private long threadId;
+    private String cpuUsed = null;
```

Comment:
I don't have strong feelings about this, but as a thought: if we keep member variables prefixed with `m` we can avoid having to disambiguate variables with `this`... your call

Reply:
@paulkagiri You can either implement or decline; if you implement feel free to resolve. If declining, just say so and resolve

Reply:
I already addressed this @iambmelt -please note there's no usage of `this` in this method - https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/neagarw/perftestcase/common/src/main/java/com/microsoft/identity/common/CodeMarker.java#L69-L88

Reply:
this will only be used on the initialization of the fields